File: test/java/negatives/F5.java
import java.util.*;

public class F5 {
    public static List<Integer> extractAndSort(List<Map<String, Integer>> inputList) {
        List<Integer> extracted = new ArrayList<>();
        for (Map<String, Integer> item : inputList) {
            if (item.containsKey("value")) {
                extracted.add(item.get("value"));
            }
        }
        extracted.sort(Comparator.reverseOrder());
        return extracted;
    }
}

File: test/python/negatives/f5.py
def extract_and_sort(input_list):
    # input_list: List of dicts, e.g., [{"value":5},{"value":2},...]
    extracted = []
    for item in input_list:
        if "value" in item:
            extracted.append(item["value"])
    extracted.sort(reverse=True)
    return extracted
